central/group/datastore/internal/store/store_impl.go

```diff
@@ -1,316 +0,0 @@
-package store
```

Comment:
Despite what GitHub shows, this file wasn't delete. It's just moved into the bolt folder and extensively modified.